sbt-plugin/src/main/scala/kalix/sbt/KalixPlugin.scala

```diff
@@ -153,4 +171,20 @@ object KalixPlugin extends AutoPlugin {
       .toArray(new Array[File](_))
       .toVector
   }
+
+  private def dockerComposeUp(file: String): Unit = {
+    import scala.sys.process._
+    import scala.concurrent.ExecutionContext.Implicits.global
+
+    // TODO: check if file exists
+    Future {
+      s"docker-compose -f $file up".!
```

Comment:
It turns out that calling `docker-compose` directly makes it much simpler as we don't need to wrap anything with testcontainers. 

If we decide to take that directly, I will refactor the `mvn kalix:run` to do exactly the same.

Reply:
Is the `docker-compose` available when installing docker or is this sth that must be installed separately?

Reply:
I also have the same question as Andrzej. I _think_ you need to install the plugin but I'm not sure if that's only the case for compose V2. That's also another question, if we need to worry about [V1 vs V2](https://docs.docker.com/compose/compose-v2/) since we have been relying on V1 but until now that was clear (the user had to go through the README and explicitely do `docker-compose up` so they could easily switch to `docker compose` if that's their case.

With this, I think we might need to check for the availability of "docker-compose" or "docker compose" and warn the user when that is not found since the use of docker-compose is much more hidden.

Reply:
yep, I would also go for checking what is installed and then warning. 

In any case, users need to have one or the other anyway. If they don't have and can't install it, they won't be able to run `sbt runAll` or `mvn kalix:runAll`. Actually, they won't be able to run their application locally anyway.

Reply:
For the record, I removed all docker binaries from an old machine and re-installed it. `docker compose` is just included in the new installation. Also `docker-compose` or maybe it works just like with git. If there is a bin named `docker-compose` you can call it without the `-`.

Reply:
some more investigations...

I have two sym-link files on my machines: `docker-compose` and `docker-compose-v1`. 

When checking the version, I get:

```shell
!❯ docker-compose-v1 --version
docker-compose version 1.29.2, build 5becea4c
```

```shell
=❯ docker-compose --version
Docker Compose version v2.12.2
```

```shell
=❯ docker compose version
Docker Compose version v2.12.2
```

I would assume that `docker-compose` is the one we should use. Doesn't matter if v1 or v2, it will just work. 


We should however, emit a warning if command fails and point user to the right place.

Reply:
https://docs.docker.com/compose/

"From the end of June 2023 Compose V1 won’t be supported anymore and will be removed from all Docker Desktop versions.

Make sure you switch to [Compose V2](https://docs.docker.com/compose/compose-file/) with the docker compose CLI plugin or by activating the Use Docker Compose V2 setting in Docker Desktop. For more information, see the [Evolution of Compose](https://docs.docker.com/compose/compose-v2/)."

Reply:
Sure, but the way that I understand it is that `docker-compose` works and will still work after June 2023, because `docker-compose` is an alias to `docker compose`. And that's V2. 


If we use `docker-compose` it works with V1 and with V2.